src/gui/advancedsettings.cpp

```diff
@@ -609,6 +617,23 @@ void AdvancedSettings::loadAdvancedSettings()
     // Announce to all tiers
     m_checkBoxAnnounceAllTiers.setChecked(session->announceToAllTiers());
     addRow(ANNOUNCE_ALL_TIERS, tr("Always announce to all tiers"), &m_checkBoxAnnounceAllTiers);
+
+    // Peer Turnover
+    m_spinBoxPeerTurnover.setMinimum(0);
+    m_spinBoxPeerTurnover.setMaximum(100);
+    m_spinBoxPeerTurnover.setValue(session->peerTurnover());
+    m_spinBoxPeerTurnover.setSuffix(tr(" %"));
+    addRow(PEER_TURNOVER, tr("Percentage of peers to disconnect"), &m_spinBoxPeerTurnover);
+    m_spinBoxPeerTurnoverCutoff.setMinimum(0);
+    m_spinBoxPeerTurnoverCutoff.setMaximum(100);
+    m_spinBoxPeerTurnoverCutoff.setSuffix(tr(" %"));
+    m_spinBoxPeerTurnoverCutoff.setValue(session->peerTurnoverCutoff());
+    addRow(PEER_TURNOVER_CUTOFF, tr("Connection threshold percentage"), &m_spinBoxPeerTurnover);
```

Comment:
If we're following the suggested naming scheme above, perhaps ""Peer turnover cutoff percentage"?

Reply:
Can do - just trying to get this description to make more sense than the LibTorrent documentation. ;-)